.flake8

```diff
@@ -1,5 +1,4 @@
 [flake8]
 max-line-length = 120
-format=abspath
```

Comment:
with that line I had got a stupid "abspath" line instead of real path:

![image](https://user-images.githubusercontent.com/13851207/105687021-8768f580-5f2a-11eb-9a29-3e16b1615041.png)